core/survey/_surveyValidator/nodeDefValidator.js

```diff
@@ -102,14 +102,14 @@ const validateAdvancedProps = async (survey, nodeDef) => {
   return Validation.newInstance(
     R.all(Validation.isValid, validations),
     R.reject(Validation.isValid, {
-      [NodeDef.propKeys.defaultValues]: validations[0],
-      [NodeDef.propKeys.applicable]: validations[1],
-      [NodeDef.propKeys.validations]: validations[2],
+      [NodeDef.keysPropsAdvanced.defaultValues]: validations[0],
```

Comment:
Perhaps the preceding lines could deconstruct the `Promise.all` invocation directly, like `const [defaultValue, applicable, validations] = Promise.all(...)`

Reply:
yes, good catch